src/NSwag.CodeGeneration.TypeScript/Models/TypeScriptFileTemplateModel.cs

```diff
@@ -164,6 +164,9 @@ public IEnumerable<string> ResponseClassNames
         /// <summary>Gets a value indicating whether the target TypeScript version supports override keyword.</summary>
         public bool SupportsOverrideKeyword => _settings.TypeScriptGeneratorSettings.SupportsOverrideKeyword;
 
+        /// <summary>Gets a value indicating whether the target TypeScript version supports Type-Only imports</summary>
+        public bool SupportsTypeOnlyImports => _settings.TypeScriptGeneratorSettings.TypeScriptVersion >= 4.3m;
```

Comment:
I think you should check against 3.8 because it's the version that supports it. User can define free value in JSON configuration,  it's only config ui that has fixed options.

Reply:
Thanks for your reply. Changed to 3.8. See commit: https://github.com/RicoSuter/NSwag/pull/4005/commits/a2c128053865b82113527abb75587c33b19b0080